src/main/java/net/sf/jabref/model/entry/Author.java

```diff
@@ -38,20 +38,100 @@
      * @param von       the von part of the author's name (may consist of several
      *                  tokens, like "de la" in "Charles Louis Xavier Joseph de la
      *                  Vall{\'e}e Poussin")
-     * @param last      the lats name of the author (may consist of several
+     * @param last      the last name of the author (may consist of several
      *                  tokens, like "Vall{\'e}e Poussin" in "Charles Louis Xavier
      *                  Joseph de la Vall{\'e}e Poussin")
      * @param jr        the junior part of the author's name (may consist of
      *                  several tokens, like "Jr. III" in "Smith, Jr. III, John")
      */
     public Author(String first, String firstabbr, String von, String last, String jr) {
-        firstPart = removeStartAndEndBraces(first);
+        firstPart = addDotIfAbbreviation(removeStartAndEndBraces(first));
         firstAbbr = removeStartAndEndBraces(firstabbr);
         vonPart = removeStartAndEndBraces(von);
         lastPart = removeStartAndEndBraces(last);
         jrPart = removeStartAndEndBraces(jr);
     }
 
+    public static String addDotIfAbbreviation(String name) {
```

Comment:
Ahh, I know the answer for that... Somehow I didn't see StringUtil in the diff the first time. :-)